studio/components/to-be-cleaned/DateRangePicker.tsx

```diff
@@ -187,27 +195,25 @@ const DateRangePicker = ({
 
   return (
     <>
-      <Dropdown
-        side="bottom"
-        align="start"
-        overlay={
-          <>
-            <Dropdown.RadioGroup value={timePeriod} onChange={(x) => handleChange(x)}>
-              {options.map((option: any) => {
-                return (
-                  <Dropdown.Radio value={option.key} key={option.key}>
-                    {option.label}
-                  </Dropdown.Radio>
-                )
-              })}
-            </Dropdown.RadioGroup>
-          </>
-        }
-      >
-        <Button asChild type="default" iconRight={<IconChevronDown />}>
-          <span>{timePeriod && options.find((x: any) => x.key == timePeriod)?.label}</span>
-        </Button>
-      </Dropdown>
+      <DropdownMenu_Shadcn_>
+        <DropdownMenuTrigger_Shadcn_>
+          <Button asChild type="default" iconRight={<IconChevronDown />}>
+            <span>{timePeriod && options.find((x: any) => x.key == timePeriod)?.label}</span>
```

Comment:
nit: `==`

Also any way we can get rid of the `: any`? 👀

Reply:
maybe not part of this refactor ?